dsc_lib/src/dscresources/command_resource.rs

```diff
@@ -6,10 +6,30 @@ use serde_json::Value;
 use std::{collections::HashMap, process::Command, io::{Write, Read}, process::Stdio};
 use crate::dscerror::DscError;
 use super::{dscresource::get_diff,resource_manifest::{ResourceManifest, InputKind, ReturnKind, SchemaKind}, invoke_result::{GetResult, SetResult, TestResult, ValidateResult, ExportResult}};
-use tracing::{debug, info};
+use tracing::{debug, info, trace};
 
 pub const EXIT_PROCESS_TERMINATED: i32 = 0x102;
 
+
+pub fn print_resource_traces(stderr: &String)
+{
+    if !stderr.is_empty()
+    {
+        for trace_line in stderr.lines() {
+            match serde_json::from_str::<Value>(trace_line){
```

Comment:
I think we should have structs defined for trace, debug, info and try to deserialize the output and do the matching logging when it deserializes otherwise we can debug an unknown trace with the unexpected output.

Reply:
As discussed, we can do this later and probably align with the tracing crate JSON format to make it easier for ourselves

Reply:
Attempting several deserializations on the same string is a big perf hit.
It is better to deserialize once and then check if the first member is Error/Warning/Debug/etc... and log accordingly.